cpu/atmega_common/atmega_cpu.c

```diff
@@ -89,7 +89,7 @@ ISR(BADISR_vect)
 {
     avr8_reset_cause();
 
-#if defined(CPU_ATMEGA256RFR2)
+#if defined(IRQ_STATUS1)
```

Comment:
The goal is to be more inclusive so I suggest change as below to in this case.

```diff
+#if defined(TRX_CTRL_0) /* megaRF */
+    printf("IRQ_STATUS %#02x\n", (unsigned int)IRQ_STATUS);
+
+#if defined(IRQ_STATUS1)
+    printf("IRQ_STATUS1 %#02x\n", (unsigned int)IRQ_STATUS1);
+#endif

    printf("SCIRQS %#02x\nBATMON %#02x\n", (unsigned int)SCIRQS, (unsigned int)BATMON);

    printf("EIFR %#02x\nPCIFR %#02x\n", (unsigned int)EIFR, (unsigned int)PCIFR);
#endif
```

Reply:
Done